### PR TITLE
Correct image registry handling

### PR DESCRIPTION
**Pull request by gchazot**
## Purpose of PR
This fixes a problem introduced with https://github.com/EDITD/kubetools/pull/141 where images that are specified directly, without a registry (hence should be assumed to be sourced from the docker daemon default registry, usually dockerhub) got modified into using the value from `--default-registry` option. That option is instead intended to point at the registry where to push the images built by kubetools and hence where k8s can pull those images from.

### CODE REVIEW

**Review by DilaraOflaz — APPROVED** [review]

File: tests/configs/docker_registry/Dockerfile-direct [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM python:3.9-slim
+ENTRYPOINT ["python"]
```
Comment (ryan109) [inline tests/configs/docker_registry/Dockerfile-direct]: Why do we need 2 test configs that are identical? The other being `Dockerfile-context`

**Review by ryan109 — APPROVED** [review]
Changes LGTM

File: tests/configs/docker_registry/Dockerfile-direct (hunk 1 above)
Comment (gchazot) [reply tests/configs/docker_registry/Dockerfile-direct]: It's just to make sure they are distinguishable in the generated k8s manifests

Comment (gchazot) [reply tests/configs/docker_registry/Dockerfile-direct]: Actually you're right here, simplifying: 1d23331

**Comment by gchazot** [comment]
Thanks for the reviews @DilaraOflaz @ryan109

**Pull request merged**